app/controllers/general.php

```diff
@@ -313,6 +313,27 @@
             throw new AppwriteException('Project not found', 404, AppwriteException::PROJECT_NOT_FOUND);
         }
 
+        // TODO: Is this broken?
+        /*
+        appwrite  | array(5) {
+        appwrite  |   [0]=>
+        appwrite  |   string(8) "role:all"
+        appwrite  |   [1]=>
+        appwrite  |   string(11) "role:member"
+        appwrite  |   [2]=>
+        appwrite  |   string(25) "user:62a0ceadbcd90be6ebaf"
+        appwrite  |   [3]=>
+        appwrite  |   string(25) "team:62a0ceb229130190e2bb"
+        appwrite  |   [4]=>
+        appwrite  |   string(31) "team:62a0ceb229130190e2bb/owner"
+        appwrite  | }
+        appwrite  | string(6) "member"
+
+        appwrite  | [Error] Message: matej@appwrite.io (role: ) missing scope (users.read)
+        */
+        \var_dump($roles);
+        \var_dump($role);
```

Comment:
https://github.com/appwrite/appwrite/blob/9cb315d272d12ca3d6c8bf1db91611ab11298d7b/app/controllers/general.php#L301

this breaks it - just rename the variable or not use it at all and it works again 👍🏻